src/content/reference/react-dom/server/renderToPipeableStream.md

```diff
@@ -4,7 +4,7 @@ title: renderToPipeableStream
 
 <Intro>
 
-`renderToPipeableStream` renders a React tree to a pipeable [Node.js Stream.](https://nodejs.org/api/stream.html)
+`renderToPipeableStream` me-*render* pohon React menjadi [Node.js Stream](https://nodejs.org/api/stream.html) yang *pipeable*.
```

Comment:
Let's introduce it once along with the original word.

```suggestion
`renderToPipeableStream` me-*render* pohon (*tree*) React menjadi [Node.js Stream](https://nodejs.org/api/stream.html) yang *pipeable*.
```

Reply:
Hmmm, how about keep it as it is for now?

Reply:
- `*pipeable*` to keep
- `pohon` still need to be introduced with `*tree*`